File: newsRS_service/src/main/java/cn/hncu/service/ManagerService.java
package cn.hncu.service;

import cn.hncu.entity.PageResult;
import cn.hncu.pojo.Manager;
import cn.hncu.pojo.Userinfo;

/**
 * Created by Enzo Cotter on 2019/5/14.
 */
public interface ManagerService {
    Manager findByUsername(String username);

    PageResult findPage(int currentPage, int pageSize, String keywords);

    Manager findOne(Integer id);

    void add(Manager manager);

    void delete(Integer id);

    void update(Manager manager);
}

File: news_manager_web/src/main/webapp/js/service/managerService.js
app.service('managerService',function ($http) {
   this.findPage = function (currentPage,pageSize,keywords) {
       return $http.get('../manager/findPage.do?currentPage='+currentPage+'&pageSize='+pageSize+'&keywords='+keywords);
   }

    this.findOne = function (id) {
        return $http.get('../manager/findOne.do?id='+id);
    }

    this.add = function (entity) {
        return $http.post('../manager/add.do',entity)
    }

    this.delete = function (id) {
        return $http.get('../manager/delete.do?id='+id);
    }
});